src/styles/blocks.scss/current-events.scss

```diff
@@ -0,0 +1,108 @@
+.current-events {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  @include onDesktop {
+    display: grid;
+    align-items: center;
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  &__foto {
+    display: flex;
+    justify-content: center;
+
+    @include onDesktop {
+      grid-column: 1 / 8;
+    }
+  }
+
+  &__foto-item {
+    width: 100%;
+    height: 100%;
+
+    margin-bottom: 20px;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__content {
+    @include onDesktop {
+      grid-column: 9 / 13;
+    }
+  }
+
+  &__exhibition {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 15px;
+
+    &-link {
+      margin-right: 30px;
+
+      text-decoration: none;
+      font-weight: 500;
+      font-size: 16px;
+      color: #687480;
+    }
+
+    &-date {
+      font-weight: 700;
+      font-size: 16px;
+      color: #1a5a4c;
+    }
+
```

Comment:
Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.